views/institutional/components/benefits/benefit-card/index.tsx

```diff
@@ -0,0 +1,33 @@
+import { Box, Typography } from '@interest-protocol/ui-kit';
+import { FC } from 'react';
+
+import { BenefitCardProps } from './benefit-card.types';
+
+const BenefitCard: FC<BenefitCardProps> = ({ title, description, Icon }) => (
+  <Box
+    p="2.5rem"
+    display="flex"
+    height="27rem"
+    borderRadius="2rem"
+    flexDirection="column"
+    border="2px solid #FFF"
```

Comment:
border here @MarioBatalha

Reply:
Yes, take a look on figma. I'm not sure if is border or box-shadow.